### ISSUE TITLE
Considering snap store for the new GO CLI

### ISSUE THREAD

**Issue by PaulRosset** [supabase/cli#50]
# Feature request

I wanted to know if u are considering storing the new CLI binary file as ["snap"](https://snapcraft.io/store) for people using Linux.

Considering that, it will be as easy to install as a `brew install`.

**Comment by soedirgo** [comment]
Thanks for the suggestion! Looks like it's possible to publish to snapcraft from [GoReleaser](https://goreleaser.com/customization/snapcraft/), so this shouldn't be a problem.

**Comment by soedirgo** [comment]
I tried setting up snapcraft to with with GoReleaser - basically the requirement on my side is for it to be fuss-free and for Actions to be able to publish to the Snap Store without intervention.

I stopped when I found that snaps with `classic` confinement has to be manually approved by Snap Store's reviewers. The CLI interacts with the Docker daemon and remote Postgres dbs so a `strict` confinement with no additional permissions wouldn't work. I still think it's possible to publish snaps, but I'll leave it to someone who are familiar with snap to build a POC of how this could with with GoReleaser on Actions.

Instead, I added Linux packages to the Releases (i.e. `.deb` etc.) although I admit this is only slightly better than binaries. 🙏
